algorithm/WebRelated/preventPaste/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Prevent Paste</title>
+</head>
+
+<body>
+    <h1>Pastekan nama anda di inputan ini</h1>
+    <label for="nama">Nama: </label>
+    <input type="text" name="nama" id="nama">
+</body>
+
+<script>
+    const input = document.querySelector("input");
+    input.addEventListener("paste", function (e) {
```

Comment:
Dikasih jarak penulisannya

```suggestion
    const input = document.querySelector("input");
    
    input.addEventListener("paste", function (e) {
```

Reply:
Oke, sudah saya perbaiki